framework/src/main/java/org/checkerframework/common/basetype/BaseTypeValidator.java

```diff
@@ -471,6 +472,14 @@ protected void visitTypeParameterBounds(
       case TYPE_PARAMETER:
```

Comment:
The Javadoc for the containing method twice says "type parameter tree" and once says "parameter type tree".  Please make them consistent.